simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/StationSimulatorFunctionalTest.java

```diff
@@ -637,6 +638,29 @@ void shouldGetHeartbeatIntervalWithGetVariablesRequest() {
         });
     }
 
+    @Test
+    void shouldSendNotifyReportOnGetBaseReportRequest() {
+        mockSuccessfulGetBaseReportAnswer();
+
+        stationSimulatorRunner.run();
+
+        GetBaseReportRequest request = new GetBaseReportRequest().withRequestId(200).withReportBase(FULL_INVENTORY);
+
+        String payload = new Call(UUID.randomUUID().toString(), ActionType.GET_BASE_REPORT, request).toJson();
+
+        await().untilAsserted(() -> {
+            List<Call> stationCalls = server.getReceivedCalls(STATION_ID);
+            assertThat(stationCalls.stream()).anyMatch(call -> call.getActionType() == ActionType.BOOT_NOTIFICATION);
+        });
+
+        stationSimulatorRunner.getStation(STATION_ID).sendMessage(new StationMessage(STATION_ID, StationMessage.Type.OCPP_MESSAGE, payload));
+
+        await().untilAsserted(() -> {
+            List<Call> stationCalls = server.getReceivedCalls(STATION_ID);
+            assertThat(stationCalls.stream()).anyMatch(call -> call.getActionType() == ActionType.NOTIFY_REPORT);
```

Comment:
I'd check that server received a set of notifyreport requests, where all requests have same requestId, then would sort them by seqId and verify tbc values

Reply:
done